specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putMachineLearningServicesQuotaRequestDedicated.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.MachineLearningServices/locations/eastus",
+    "resourceName": "StandardDv2Family",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "unit": "Count",
+        "name": {
+          "value": "StandardDv2Family"
+        },
+        "resourceType": "dedicated"
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "StandardDv2Family",
+        "type": "Microsoft.Quota/ServiceLimits",
+        "properties": {
+          "provisioningState": "InProgress",
+          "message": ""
+        }
+      }
+    },
+    "200": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "StandardDv2Family",
+        "type": "Microsoft.Quota/serviceLimits",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "message": "Request succeeded",
+          "properties": {
+            "properties": {
```

Comment:
The schema for the request and the response should be the same. Please remove it.

Reply:
I will update the example.

Reply:
Updated the schema to return CurrentQuotaLimitBase, same as request-
"responses": {
          "200": {
            "description": "OK. Returns the quota request details.",
            "schema": {
              "$ref": "#/definitions/CurrentQuotaLimitBase"
            }
          },
          "201": {
            "description": "The request is accepted and is being processed. To check the status for this request, use the value for the quota request ID **id** obtained in the response of a [Quota Request Status](https://docs.microsoft.com/en-us/rest/api/reserved-vm-instances/quotarequeststatus/get) GET operation.",
            "schema": {
              "$ref": "#/definitions/CurrentQuotaLimitBase"
            }
          }